openapi/media_type.go

```diff
@@ -8,20 +8,31 @@ type MediaType struct {
 	Schema *Schema
 }
 
-func (m *MediaType) GetName() string {
-	return m.parent.GetName() + m.name
-}
+//func (m *MediaType) GetName() string {
+// TODO where is this used?
+//
+//	return m.parent.GetName() + "[" + m.node.name + "]"
+//}
```

Comment:
This method overrides the default node implementation, because I didn't want "application/json" littered all over generated templates' model names, since we don't support other media types at the moment.

I should have added that as a comment. My bad. Can we keep this in its old form please? If you would like to make this more clear, consider adding a comment to explain why its there.

Reply:
Done, comment added